Makefile

```diff
@@ -640,6 +650,8 @@ test_integrations_mysqli:
 test_integrations_mongo:
 	$(MAKE) test_scenario_default
 	$(call run_tests,tests/Integrations/Mongo)
+test_integrations_pcntl:
+	$(call run_tests,tests/Integrations/PCNTL)
```

Comment:
I don't think this is running in CI. Do we need to add these to `TEST_INTEGRATIONS_*`?

Reply:
Good catch. addressed in https://github.com/DataDog/dd-trace-php/pull/1196/commits/b0dd94c4aa3add2e3b8ffb2f187606361060bca5